x-pack/plugins/endpoint/common/types.ts

```diff
@@ -22,7 +24,7 @@ export type ImmutableMap<K, V> = ReadonlyMap<Immutable<K>, Immutable<V>>;
 export type ImmutableSet<T> = ReadonlySet<Immutable<T>>;
 export type ImmutableObject<T> = { readonly [K in keyof T]: Immutable<T[K]> };
 
-export type Maybe<T> = T | void;
+export type Maybe<T> = T | undefined;
```

Comment:
@peluja1012 Isn't this essentially the same, with strict null checks on? School me, hehe.